browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -180,9 +180,27 @@ export class CommentSection extends CanvasSectionObject {
 		}
 	}
 
-	public static showCommentEditingWarning (): void {
-		window.L.Map.THIS.uiManager.showInfoModal('annotation-editing', _('A comment is being edited'),
-		_('Please save or discard the comment currently being edited.'), null, _('Close'));
+	public navigateAndFocusComment(annotation: any): void {
+		this.scrollCommentIntoView(annotation);
```

Comment:
If you want to distinguish the behavior in calc/impress vs wrtier, here is a good place.
`map.getDocType()` is what you need